wolfcrypt/src/asn.c

```diff
@@ -15999,6 +16005,14 @@ int DecodeECC_DSA_Sig(const byte* sig, word32 sigLen, mp_int* r, mp_int* s)
         return ASN_ECC_KEY_E;
     }
 
+#ifndef NO_STRICT_ECDSA_LEN
```

Comment:
Maybe this could use a comment on why this check (in addition to the `sigLen != idx + (word32)len` check and beginning of the function) is necessary?